README.md

```diff
@@ -670,16 +670,16 @@ the `_artifactsLocation` parameter of the template to point to a specific tagged
 
 **Targeting a specific template version is recommended for repeatable production deployments.**
 
-For example, to target the [`7.0.0` tag release with PowerShell](https://github.com/elastic/azure-marketplace/tree/7.0.0)
+For example, to target the [`7.4.0` tag release with PowerShell](https://github.com/elastic/azure-marketplace/tree/7.4.0)
```

Comment:
7.5.0?

Reply:
Doesn't need to be as it's just an example, but have updated

Reply:
Doesn't need to be as it's just an example, but have updated

Reply:
++ only version 7.4.2 exists, I would just bump to latest always.